src/schema/comments.ts

```diff
@@ -628,6 +630,7 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
 
             return builder;
           },
+          readReplica: true,
```

Comment:
For postComments just to make sure we do client side optimistic vs refetching postComments because that would mean read replica might not have a comment change yet.